SimNet-Guide.md

```diff
@@ -0,0 +1,227 @@
+
+`xud` is in alpha stage and the `xud-simnet` aims to give exchange operators and testers an early 'look & feel' of how things will work. `xud` in it's current stage should never be connected to a production system or be configured for mainnet usage. The `xud-simnet` uses test coins at all times.
+
+Once the setup of `xud-simnet` is completed, you will be able to query pending orders, place orders, and experiment with `xud`'s rich set of commands. When a match is found in the network, your orders will automatically be executed via cross-chain atomic swaps. `xud-simnet` currently supports atomic swaps between BTC and LTC. This guide will show how to install dependencies like Go and how to use `xud-simnet` to automatically setup bitcoin and litecoin daemons, lightning daemons (LND) for bitcoin (BTC) and litecoin (LTC), `xud` and finally connect to the `xud-simnet`. This guide is written for Linux & MacOS and suitable for everyone that knows how to use a command line.
+
+## Describing the Setup
+
+`xud-simnet` spins up several components on your machine which together provide support for exchanging order information and executing atomic swaps between BTC and LTC on the lightning network. It will setup the following components:
+
+- `BTCD` - full node, connected to the BTC chain
+- `LTCD` - full node, connected to the LTC chain
+- `LND-BTC` - lightning network daemon for the BTC network
+- `LND-LTC` - lightning network daemon for the LTC network
+- `XUD` - Exchange Union's decentralized exchange layer
+
+## Installation 
+
+### Prerequisites
+
+Make sure you have the below programs installed:
+- [git](https://gist.github.com/derhuerst/1b15ff4652a867391f03#file-linux-md)
+- [make](https://www.cyberciti.biz/faq/debian-linux-install-gnu-gcc-compiler/)
+- [node.js + npm](https://nodejs.org/en/download/package-manager/#debian-and-ubuntu-based-linux-distributions) (use node 10 - we will drop support for node 8 soon)
+- python 2.7 (preinstalled on most systems, check with `python --version`)
+- killall (`sudo apt-get install psmisc`)
+
+### Installing Go
+
+If Go is not yet installed on your system, use the following command to install it:
+
+```
+sudo apt-get install golang-1.10-go
+```
+
+If Go is not included in your package manager, follow the official [install instructions](https://golang.org/doc/install). Note that `golang-1.10-go` puts binaries in `/usr/lib/go-1.10/bin`. If you want them on your PATH, you need to make that change yourself. Alternatively, you can create a softlink with
+
+```
+sudo ln -s /usr/lib/go-1.10/bin/go /usr/local/bin/go 
+```
+
+### Repository Checkout
+
+```
+git clone https://github.com/ExchangeUnion/xud-simnet.git ~/xud-simnet
+```
+
+This should create the `xud-simnet` directory under your home directory.
+
+### Setup ~/.bashrc
+
+To enable access to the scripts please update and source your `.bashrc`:
+
+```
+source ~/xud-simnet/setup.bash
+source ~/.bashrc

```

Comment:
I would run `source ~/xud-simnet/setup.bash` after `source ~/.bashrc` to make sure the simnet environment variables override the default ones of the user

Reply:
Fixed.